agent/command/git.go

```diff
@@ -533,6 +542,25 @@ func (c *gitFetchProject) executeLoop(ctx context.Context,
 
 	//Apply patches if this is a patch and we haven't already gotten the changes from a PR
 	if evergreen.IsPatchRequester(conf.Task.Requester) && conf.GithubPatchData.PRNumber == 0 {
+		for _, patchId := range c.AdditionalPatches {
+			if bson.IsObjectIdHex(patchId) {
```

Comment:
Is it possible this won't be true?  Should that result in an error or warning?

Reply:
yep I moved the validation for this into ParseParams